src/components/chat-message.tsx

```diff
@@ -184,17 +186,30 @@ export const ChatMessage = ( {
 
 function Anchor( props: JSX.IntrinsicElements[ 'a' ] & ExtraProps ) {
 	const { href } = props;
-	const { node, ...propsSansNode } = props;
+	const { node, className, ...filteredProps } = props;
+	const { selectedSite, startServer, loadingServer } = useSiteDetails();
 
 	return (
 		<a
-			{ ...propsSansNode }
-			onClick={ ( e ) => {
+			{ ...filteredProps }
+			className={ cx(
+				className,
+				selectedSite && loadingServer[ selectedSite.id ] && 'animate-pulse duration-100 cursor-wait'
```

Comment:
This wait cursor and pulse animation communicates the background activity of starting the site is utilized in addition to the existing activity indicators displayed in the sidebar and site header. It is fairly subtle. I welcome alternative ideas for communicating this background activity.

Reply:
I like this. Nice idea!